resources/views/patpass_comercio/create_form.blade.php

```diff
@@ -31,7 +31,7 @@
     <label for="serviceId">
         Service Id (rut con digito verificador, sin puntos ni guión)
     </label>
-    <input id="serviceId" name="serviceId" value="{{ '184399792' }}"/>
+    <input id="serviceId" name="serviceId" value="{{ 'service_'.rand(10,100) }}"/>
```

Comment:
if this is now the format for serviceId, then the description label should be updated too

Reply:
it changed